assets/components/doubleHeading/doubleHeading.jsx

```diff
@@ -10,21 +10,28 @@ import React from 'react';
 type PropTypes = {
   heading: string,
   subheading: ?string,
+  modifierClass: ?string,
 };
 
 
 // ----- Component ----- //
 
 export default function DoubleHeading(props: PropTypes) {
 
+  let className = 'component-double-heading';
+
+  if (props.modifierClass) {
+    className = `${className} ${className}--${props.modifierClass}`;
+  }
```

Comment:
I feel like this code also exists elsewhere, could we abstract out into a helper?